Add database configuration document to new wiki (#5934)
aces/Loris: docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/Database Configuration.md

# Database Configuration

## Creating the Database
MySQL or MariaDB must be installed and a `root` or admin-level user 
credential must be created before continuing. (This is not the same as a unix 
        `root` credential.) 

After running the install script, you will be prompted to visit the 

> `installdb.php` page at the IP address that LORIS is hosted at.

Reviewer: It should be at the canonical domain that it's to be hosted at, the install script uses the place that it's
accessed from in order to derive `$host`

> `installdb.php` page at the IP address that LORIS is hosted at.

Author: ```suggestion
`installdb.php` page at the canonical domain of your LORIS instance.
```

Completing the form on this page will create a new database and user account
that will be used to execute transactions coming from LORIS's PHP code.

> ### Administrative Account

Author: ```suggestion
## Administrative Account
```
